packages/sdk-react/package.json

```diff
@@ -23,7 +24,11 @@
     "compile-translations": "lingui compile",
     "write-package-version-file": "yarn exec ../../write-package-version-file.sh",
     "storybook": "storybook dev -p 6006",
-    "build-storybook": "storybook build -o storybook_static"
+    "build-storybook": "storybook build -o storybook_static",
+    "download-openapi-document": "curl -o 'src/api/schema.json' \"https://api.dev.monite.com/openapi.json?version=$(node -p 'require(\"@monite/sdk-react/package.json\").apiVersion')\" && prettier --write src/api/schema.json",
```

Comment:
Should the link to the API contain a dynamic stand? Or at least let's get the schema from the sandbox. dev doesn't look like a reliable source